apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-detail-page.component.html

```diff
@@ -33,7 +33,15 @@
         </div>
         <span class="mt-1 entity-info">
           {{'VO_DETAIL.ENTITY' | translate}}, {{'VO_DETAIL.SHORT_NAME' | translate}}:
-          {{vo.shortName}}
+          {{vo.shortName}},
```

Comment:
```suggestion
          {{vo.shortName}}<span *ngIf="enrichedVo.memberVos.length !== 0 || enrichedVo.parentVos.length !== 0">, </span>
```

we don't want the "," here if there is no continuing